Lesson1/src/main/java/ru/geekbrains/lesson1/HomeWorkApp.java

```diff
@@ -59,4 +63,47 @@ public static void compareNumbers()
         else
             System.out.println("a<b");
     }
+
+    public static boolean checkSumm(int a,int b)
+    {
+        int c=a+b;
+        boolean rslt = false;
+        if ((c >=10)&&(c<=20))

```

Comment:
поскольку логическое выражение само по себе возвращает boolean результат, то можно упростить:
```suggestion
        return(c >=10) && (c<=20);

```